CHANGELOG.next.asciidoc

```diff
@@ -10,7 +10,6 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Affecting all Beats*
 
-- The document id fields has been renamed from @metadata.id to @metadata._id {pull}15859[15859]
```

Comment:
Actually I think we should expose this change because some 7.5 users might have Logstash configs that expect @metadata.id.

Reply:
I've added this line back.